.pre-commit-config.yaml

```
@@ -14,7 +14,7 @@ repos:
           - id: check-docstring-first
           - id: check-executables-have-shebangs
           - id: check-json
-            exclude: tsconfig.json
+            exclude: tsconfig.json pyscript.core/cjs/package.json
```

Does this work correctly? I would think you'd have to do:
```suggestion
            exclude: tsconfig\.json|pyscript.core/cjs/package\.json
```

---

it seems to work or CI would be red?

---

Strange, [the pre-commit docs](https://pre-commit.com/#regular-expressions) say pretty clearly that this is a regex. As a regex "A\.py B\.py" shouldn't match anything... There must be something I'm not understanding.

---

P.S. I don't mind changing the way you suggested, all I am saying here is that if it's what you think it should be, we had already a bug in that file that predates anything I've done in here

---

Yeah I understand why `tsconfig.json` works though: in a regex, `.` is a wildcard which in particular matches a literal period. It also matches other stuff like I think it would also ignore a file called `tsconfig-json.txt`. A bit weird but not the end of the world. What's confusing to me though is why the current thing works at all. But it's not very important: if pre-commit is satisfied then we can leave it alone.

---

I perfectly know RegExp inside out, my point was that it was a mistake to have just `.` before without escape and, since I've seen that and didn't know a RegExp was meant because of that dot not escaped, I've used that syntax and it just works. I don't have any reason to not apply the right escape, all I am saying is that it does work, and it was wrong already anyway.